build/lib/config.yaml

```
@@ -0,0 +1,32 @@
+
+    - name: Install dependencies
+      run: |
+        python -m venv venv
+        source venv/Scripts/activate  # Use `source venv/bin/activate` on Ubuntu
+        pip install -r requirements.txt  # Ensure mypy is in your requirements
+        pip install mypy
+
+    - name: Run mypy
+      run: |
+        source venv/Scripts/activate
+        mypy . --config-file mypy.ini  # Ensure mypy.ini is in your root directory
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)